### ISSUE TITLE
Warning issued if site_url parameter omitted

### ISSUE THREAD

**Issue by xxsimoxx** [codepotent/Update-Manager#31]
# Description 
A warning is thrown [here](https://github.com/codepotent/Update-Manager/blob/1b0552d146eea55c5b802aebe4ce2d65ae0e0cdb/includes/functions.php#L59) in the *exceptional* case (RC1) where the URL is taken from user agent.

# Steps to Reproduce
The easiest way:
View an endpoint and remove the `&site_url=https://xxxx.yyyy.it` part from the URL.

# Expected Behavior
Just `[]` should be outputted.

# Possible fix:
As `array_pop` takes a reference the line should be split in this way:
```php
$user_agent_array=explode(';', $_SERVER['HTTP_USER_AGENT']);
$site_url = trim(array_pop($user_agent_array));
```
*I've not created a PR because it seems that the repo is not at 1.0.1.*

**Comment by johnalarcon** [comment]
> I've not created a PR because it seems that the repo is not at 1.0.1.

Thanks. I tend to do a larger chunk of work and commit it all at once, so, there's no guarantee `master` will every be equivalent to any particular version. About this issue, though, the expected behavior you describe is reasonable and it will be implemented. Thanks for the report.

**Comment by johnalarcon** [comment]
The proposed fix has been implemented and will be released in version 2.0.0. If the `site_url` URL argument is missing, an empty array is returned.
